spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,48 @@
+require 'beaker-rspec'
+
+# Various workarounds for Beaker's lack of Puppet 4 support.
+
```

Comment:
I'm a bit confused about this.  If I look at any of the Puppet Labs modules, they're all using the puppet install helper.  However, if I run their tests and set BEAKER_destroy=no they're tested against Puppet 3.8.6.  Moreover I've seen in the code that all of Beaker's defaults (dist dir path for instance) are hardcoded for Puppet 3.x.  Am I missing something?